src/bootstrap/builder.rs

```diff
@@ -662,7 +663,7 @@ impl<'a> Builder<'a> {
             cargo.env("RUSTDOC_LIBDIR", self.rustc_libdir(self.compiler(2, self.config.build)));
         }
 
-        if mode == Mode::Tool {
+        if Tool::is_tool(mode) {
```

Comment:
Could we instead make this inherent, so `mode.is_tool()`? I think I may have previously mislead you; I intended this to be implemented on `Mode`.

Reply:
No worries! Going to fix this ASAP. Thanks